File: Morning Snacks/Armstrong1-1000.js
let number =1000

for (let count=1; count<=number;count++){
      let  sumOfCube =0
      let digits = count
            while ( digits!=0){
          let seperateDigits = digits%10
          let cube= seperateDigits**3
                  sumOfCube += cube
               digits= Math.floor(digits/10)
          }
            if (sumOfCube==count){
               console.log(count)
     }
}

        

File: Morning Snacks/Python/Armstrong1-1000.py
number = 1000

for count in range(1, number + 1):
    sumOfCube = 0
    digits = count
    numberLenght = len(str(count))
    
    while digits != 0:
        seperateDigits = digits % 10
        cube = seperateDigits ** numberLenght
        sumOfCube += cube
        digits = digits // 10
    
    if sumOfCube == count:
        print(count)

